src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/SearchObject.cs

```diff
@@ -88,6 +123,7 @@ public async ValueTask<SearchResultContextInfo> SearchAsync(string searchText, S
         public async ValueTask<SearchResultContextInfo> RefreshSearchAsync(CancellationToken cancellationToken)
         {
             Assumes.NotNull(_lastMainFeedSearchResult);
+            Assumes.NotNull(_lastSearchFilter);
```

Comment:
Dumb question:  if one refreshes the Browse tab with default search filter, which is no filter, will this code path be hit?  If yes, will `_lastSearchFilter` be `null`?

Reply:
This code path will not be hit, lastMainFeedSearchResult would be null first if this was going to be hit and we have checks in the other places where search filter is passed in to ensure not null.